packages/cashier/src/Components/payment-agent-list.jsx

```diff
@@ -130,8 +131,8 @@ const PaymentAgentList = ({
                                                 content: (
                                                     <PaymentAgentDetails
                                                         payment_agent_email={payment_agent.email}
-                                                        payment_agent_phone={payment_agent.phone}
-                                                        payment_agent_url={payment_agent.url}
+                                                        payment_agent_phones={toJS(payment_agent.phones)}
```

Comment:
It isn't backward compatible. We've to to let the FE code work correctly with both the old and the new backend code. Is it possible to do it like this (it's just a sketch, I'm not a JS developer)?
```
payment_agent_phones= payment_agent.phones? {toJS(payment_agent.phones)}: [{phone_number: payment_agent.phone}];
// and the same for urls
```

Reply:
@mat-fs You mean non of these fields (phone and phones) have data at all times?

Reply:
1- With the old backend code new fields are absent (`phones`, `urls` or `suppported_payment_methods`); there are only old fields (`phone`, `url`, `supported_banks`). 
2- With new new code, both new and old fields are existing (but we prefer to use the new ones), because:
3- In near future there will be only the new fields